packages/webapp/pages/onboarding.tsx

```diff
@@ -159,7 +163,13 @@ export function OnboardPage(): ReactElement {
   };
 
   const onSuccessfulRegistration = () => {
-    setIsFiltering(true);
+    if (onboardingV4dot5 === OnboardingV4dot5.Control) {
+      setIsFiltering(true);
+    } else {
+      router.replace({
+        pathname: '/my-feed',
```

Comment:
I think we need the `welcome` param here as well right?

Reply:
Not really, this is just the redirect from after signup to get to my-feed page.
After the feed has been generated, we add the welcome to their current url.

Reply:
Noticed now that if I add welcome after signup, the welcome banner is shown in the preview feed 😬 
<img width="1371" alt="image" src="https://github.com/dailydotdev/apps/assets/1681525/7cad55d1-e962-4719-8676-5dd7ec062d0b">

Reply:
Oh right, sorry missed the fact that the feed won't be ready yet 😅

Reply:
Also just did a change to hide the welcome banner if the feed is actually not ready, but the user has found the welcome url param 😬